backend/docs/SETUP_COMPOSE.md

```diff
@@ -81,3 +81,80 @@ docker-compose logs --tail 100 -f web
 docker-compose exec web bash
 
 ```
+
+
+# (Optional) Use IDEs
+
+You may want to use IDEs to boost your productivity. This session collects how we set up our IDEs.
+
+Tips: It will help a lot if you know how to set up your development via command line when you switch between different 
+IDEs. For example, from PyCharm to VS Code, or in the other way around. Make sure you understand each set-up step 
+of your IDE, and try to reproduce it with commands in command line.
+
+## PyCharm
+
+PyCharm support npm and django runners together. This fits our project very much since our project uses vue.js for 
+frontend and django for backend.
+
+### Setting npm debugger for Frontend
+
+- Run --> Edit Configurations --> + (Add New Configuration) --> npm
+- Fill in necessary information required to launch frontend
+  - tip: In "Before launch", add "Open Browser" so PyCharm will open the browser with JavaScript debugger of 
+    PyCharm. This is very handy!
+  - See [this reference](https://www.jetbrains.com/help/pycharm/run-debug-configuration-npm.html) if you need more detailed instructions
+- Use "Run" to launch frontend if you simply want to run the frontend
+- Use "Debug" to launch frontend if you want to use breakpoints to debug interactively.
+
+#### Source Entry Points
+
+Put your breakpoints in the functions you are interested in.
+
+- On creating ohshown event
```

Comment:
@Neilxx @yanghaochang104 I think you may add more information about this part :)